### DISCUSSION TITLE
Using hintAction()

### DISCUSSION THREAD

**Discussion by sinnbeck** [filamentphp/filament#7982] [Help] [answered]
I think this might be a bug, but there is a chance I am simply missing something obvious when using hintActions()

I have added a form to a regular livewire component, but when I click the hintAction() button, it loads for a second, and nothing else happens. I have copy pasted the code exactly from the docs, to make sure the code isnt in my own implementation
```php
public function form(Form $form)
{
    return $form
        ->schema([
            TextInput::make('cost')
                ->prefix('€')
                ->hintAction(
                    Action::make('copyCostToPrice')
                        ->icon('heroicon-m-clipboard')
                        ->requiresConfirmation()
                        ->action(function (Set $set, $state) {
                            $set('price', $state);
                        })
                )
        ]);
}

**Comment by danharrin** [comment, marked as answer]
Do you have `<x-filament-actions::modals />` in the view

**Reply by sinnbeck** [reply]
Ah I knew I was missing something! Thank you again
